### PR TITLE
Add source esx7.0 and change source of v2v tier2 guests to esx7.0

### PR DESCRIPTION
**Pull request by mxie91**
V2V only need to convert tier2 guests from latest ESXi host

Signed-off-by: mxie91 <mxie@redhat.com>

### CODE REVIEW

File: v2v/tests/cfg/convert_vm_to_libvirt.cfg [hunk 1]
```diff
@@ -48,26 +48,26 @@
                 - 5_11:
                     os_version = "rhel5.11"
                 - debian:
-                    only source_esx.esx_67
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_libvirt.cfg:51]: It's better keep both cases for esx6.7 and 7.0 by 'only esx_67, esx_70'.
And your changes must require some avocado-vt changes.

Comment (mxie91) [reply v2v/tests/cfg/convert_vm_to_libvirt.cfg:51]: v2v only need to convert these tier2 guests from latest ESXi in v2v matrix testing, if keep esx_67 source, these guests will be converted from esx6.7 in matrix testing too.

Already submit https://github.com/avocado-framework/avocado-vt/pull/2556 to avocado-vt, pls review

**Comment by mxie91** [comment]
# avocado run --vt-type v2v convert_vm_to_libvirt.esx.vm.7_0.linux.debian.arch_x86_64.it_vddk
JOB ID     : ae4665930ff9d2c77ca3cb534569e8763b5ffe37
JOB LOG    : /root/avocado/job-results/job-2020-05-14T03.51-ae46659/job.log
 (1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.7_0.linux.debian.arch_x86_64.it_vddk: PASS (378.96 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 380.02 s

# avocado run --vt-type v2v convert_vm_to_libvirt.esx.vm.7_0.linux.ubuntu_lts.arch_x86_64.it_vddk
JOB ID     : e0100513068cf66104e96b883bc914fc48cbc21c
JOB LOG    : /root/avocado/job-results/job-2020-05-14T04.02-e010051/job.log
 (1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.7_0.linux.ubuntu_lts.arch_x86_64.it_vddk: PASS (886.54 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 887.69 s

# avocado run --vt-type v2v convert_vm_to_libvirt.esx.vm.7_0.linux.opensuse15_1.arch_x86_64.it_vddk
JOB ID     : 8a9331f30c36034dc26d7d5cfb2f4c5b6511b76d
JOB LOG    : /root/avocado/job-results/job-2020-05-14T06.04-8a9331f/job.log
 (1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.7_0.linux.opensuse15_1.arch_x86_64.it_vddk: PASS (854.61 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 855.71 s

# avocado run --vt-type v2v convert_vm_to_libvirt.esx.vm.7_0.linux.opensuse_latest.arch_x86_64.it_vddk 
JOB ID     : 02c1c1e33133620d69921582fa0ed4b7fb342ee7
JOB LOG    : /root/avocado/job-results/job-2020-05-14T06.52-02c1c1e/job.log
 (1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.7_0.linux.opensuse_latest.arch_x86_64.it_vddk: PASS (820.14 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 821.20 s

File: v2v/tests/cfg/convert_vm_to_libvirt.cfg (hunk 1 above)
Comment (xiaodwan) [reply v2v/tests/cfg/convert_vm_to_libvirt.cfg:51]: I don't think so, You can select the cases by specifying appropriate case pattern in libvirt-ci.

**Comment by xiaodwan** [comment]
depends on 'https://github.com/avocado-framework/avocado-vt/pull/2556'

File: v2v/tests/cfg/convert_vm_to_libvirt.cfg (hunk 1 above)
Comment (yantian0) [reply v2v/tests/cfg/convert_vm_to_libvirt.cfg:51]: Agree with keeping both cases for esx6.7 and 7.0.

Comment (mxie91) [reply v2v/tests/cfg/convert_vm_to_libvirt.cfg:51]: Hi xiaodwan, how to define the pattern to make sure v2v won't convert these tier2 guests from esx6.7 if keep esx6.7 source, could you please give me a suggestion? Pls note these tier2 guests are defined in same variants 'linux 'with rhel guests and current pattern in matrix testing is like below:
convert_vm_to_libvirt.esx.vm.6_7..it_default
convert_vm_to_ovirt.esx.vm.6_7..NFS.rhv.it_default

Comment (xiaodwan) [reply v2v/tests/cfg/convert_vm_to_libvirt.cfg:51]: convert_vm_to_libvirt.esx.vm.6_7  will run all guests on esx6.7.
convert_vm_to_libvirt.esx.vm.7_0 will run all guests on esx7.0.

Comment (xiaodwan) [reply v2v/tests/cfg/convert_vm_to_libvirt.cfg:51]: I think you can use 'ci-no' to filter out the cases you don't want to run.

File: v2v/tests/cfg/convert_vm_to_libvirt.cfg [hunk 2]
```diff
@@ -48,26 +48,26 @@
...
+                    only source_esx.esx_67, only source_esx.esx_70
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_libvirt.cfg:51]: IIRC, You should not add a 'only' after ','.

File: v2v/tests/cfg/convert_vm_to_ovirt.cfg [hunk 3]
```diff
@@ -88,22 +88,22 @@
...
-                    only source_esx.esx_67
+                    only source_esx.esx_67, only source_esx.esx_70
                     os_version = "DEBIAN_VERSION"
                 - ubuntu_lts:
-                    only source_esx.esx_67
+                   only source_esx.esx_67, only source_esx.esx_70
                     os_version = "UBUNTU_LTS_VERSION"
                 - sles12sp4:
-                    only source_esx.esx_67
+                  only source_esx.esx_67, only source_esx.esx_70
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_ovirt.cfg:97]: The indention is not 4 blanks aligned.

Comment (mxie91) [reply v2v/tests/cfg/convert_vm_to_ovirt.cfg]: fixed, pls check again

**Pull request merged**